static/game/block.js

```diff
@@ -37,6 +38,7 @@ export function destroyBlock(cellVec, grantPoints = true) {
     if (grantPoints) {
       const gemType = blockField[cellVec.x][cellVec.y].gemType;
       if (gemType !== undefined) {
+        playSound("gem");
```

Comment:
maybe different sound effects for different gems?

Reply:
i like that idea